packages/terra-icon/scripts/src/icon-utils/parseCsv.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-extraneous-dependencies, compat/compat */
+import csv from 'csvtojson';
+import _ from 'lodash';
+import { TerraIcon } from '../config';
+/* eslint-enable import/no-extraneous-dependencies */
+
+const csvHeaders = ['name', 'filepath', 'themeable', 'bidi'];
+
+const parseCsv = () => new Promise((resolve, reject) => {
```

Comment:
Lifted this code from [this file](https://github.com/cerner/terra-core/blob/master/packages/terra-icon/scripts/src/generate-icon/parseSvg.js) into a new util file as it is now used in the generate-icons scripts and generate-json scripts.